MyHeartCounts/MyHeartCountsStandard+Preview.swift

```diff
@@ -0,0 +1,26 @@
+//
+// This source file is part of the My Heart Counts iOS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import Spezi
+import SpeziAccount
+import SpeziStudy
+
+
+extension MyHeartCountsStandard {
+    @MainActor @ModuleBuilder static var previewModels: ModuleCollection {
+        FirebaseConfiguration()
```

Comment:
we have a `DeferredFirebaseLoading.baseModules` (or similar) property which is an array of the always-enabled base modules; might make sense to use that here?